guides/common/modules/con_managing-dhcp-using-smart-proxy.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Managing_DHCP_Using_Smart_Proxy_{context}"]
+= Managing DHCP Using {SmartProxy}
+
+{Project} can integrate with a DHCP service using {SmartProxies}.
+A {SmartProxy} has multiple providers, allowing you to integrate with your existing DHCP infrastructure or deploy a new one.
+You can use the DHCP module of {SmartProxy} to query for available IP addresses, add new, and delete existing reservations.
+Note that your {SmartProxy} cannot manage subnet declarations.
+
+The DHCP module can manage various available DHCP providers:
```

Comment:
"implements" instead of "can manage"?

Reply:
Or maybe just a phony heading like `.Available DHCP Providers` instead of this sentence? (same as for DNS below)